main.go

```diff
@@ -74,7 +74,6 @@ func main() {
 	ircListenerPrejoinCommands := viper.GetStringSlice("irc_listener_prejoin_commands") // Commands for each connection to send before joining channels
 	guildID := viper.GetString("guild_id")                                              // Guild to use
 	webIRCPass := viper.GetString("webirc_pass")                                        // Password for WEBIRC
-	identify := viper.GetString("nickserv_identify")                                    // NickServ IDENTIFY for Listener
```

Comment:
One last thing, sorry for not mentioning it earlier, could we add a thing here that exits with an error message, linking to the example config file? https://github.com/qaisjp/go-discord-irc/blob/master/config.yml That way if anyone is using this setting, they will understand they need to update their config, and know how to do it.

Reply:
Will do, will work on that after breakfast. That's a great idea!

Reply:
Just fixed. Let me know if the error message needs changing, it may be a tad verbose or wordy but I couldn't think of a way to get it to be more concise.

Reply:
Looks great :) Thank you!!!